hugo/content/lookout.md

```diff
@@ -4,4 +4,6 @@ section: lookout
 url: lookout
 title: source{d} Lookout
 socialDescription: A service for assisted code review, that allows running custom code Analyzers on pull requests.
+socialImage: img/lookout-social-card.png
```

Comment:
I'd create new logos image into [`static/img/logos`](https://github.com/src-d/landing/tree/master/static/img/logos), so they'll be all in the same dir.
```suggestion
socialImage: img/logos/lookout-social-card.png
```